<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiftMatch</class>
 <widget class="QDialog" name="SiftMatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIFT拼接</string>
  </property>
  <property name="whatsThis">
   <string>这是一个基于SIFT特征点的图片匹配和全景拼接程序。</string>
  </property>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>60</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detectButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>100</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>特征点检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="matchButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>170</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>特征匹配</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>250</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重新选取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mosaicButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>210</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全景拼接</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_horizontal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横排</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_vertical">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵排</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#5500ff;&quot;&gt;只能打开2张图片,左右顺序任意&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;路径中不能有中文&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
